src/cgame/cg_rocket_draw.cpp

```diff
@@ -2442,6 +2442,35 @@ class NumSpawnsElement : public TextHudElement
 	int spawns_;
 };
 
+class PlayersCountElement : public TextHudElement
+{
+public:
+	PlayersCountElement( const Rml::String& tag ) :
+			TextHudElement( tag, ELEMENT_GAME )
+			{}
+
+	void DoOnUpdate() override
+	{
+		//there is probably a better way to do that, maybe reusing value
+		//cached in some struct?
+		std::string teamname = GetAttribute< Rml::String >( "team", "" );
+
+		team_t team = BG_PlayableTeamFromString( teamname.c_str() );
+		size_t count = 0;
+
+		for ( int i = 0; i < cg.numScores; ++i )
+		{
+			score_t const& score = cg.scores[ i ];
+
+			if ( score.team == team )
+			{
+				++count;
+			}
+		}
+
+		SetText( va( "Player count: %zu", count ) );
```

Comment:
Just set the count, not the other text, let the UI decide whether it wants to the text there or not. (It could want to style it differently or use an image for example).